source/_integrations/water_heater.mqtt.markdown

```diff
@@ -0,0 +1,312 @@
+---
+title: "MQTT Water Heater"
+description: "Instructions on how to integrate MQTT Water Heater into Home Assistant."
+ha_category:
+  - Water Heater
+ha_release: 2023.x
```

Comment:
Release needs to made specific

Reply:
Would that be `2023.6.0` or will I have to wait until the parent is merged to see what release?

Reply:
We missed the beta cut yesterday, so that will be 2023.7.0